reth-extension/src/abci/abci_ext.rs

```diff
@@ -73,8 +75,19 @@ impl Future for AbciTask {
     type Output = ();
 
     fn poll(self: std::pin::Pin<&mut Self>, cx: &mut std::task::Context<'_>) -> Poll<Self::Output> {
+        use std::task::ready;
+        use tracing::info;
+
         let this = self.get_mut();
-        loop {}
-        Poll::Pending
+
+        loop {
+            // listen for incoming custom transactions
+            if let Some(AbciMessage::RawTransaction(tx)) = ready!(this.rx_abci.poll_recv(cx)) {
+                info!("Received raw transaction: {:?}", tx);
```

Comment:
Nên implement trước thêm phần abci request để tạo đúng request từ pool, tránh convert nhiều lần từ transaction ->  rawData (bytes) -> abci request

Reply:
we need to find the solution of which abci client we should use, and after that, we could know exactly which type of request is